src/components/LeftNavBar/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { calc, Flex, FlexProps, HeadingProps } from "@chakra-ui/react"
+
+import ButtonDropdown, {
+  IProps as ButtonDropdownProps,
+  List as ButtonDropdownList,
+} from "../ButtonDropdown"
+import OldHeading from "../OldHeading"
+import Translation from "../Translation"
+import UpgradeTableOfContents from "../UpgradeTableOfContents"
+
+export const H2 = (props: HeadingProps) => (
+  <OldHeading
+    fontWeight={700}
+    lineHeight={1.4}
+    fontSize="2xl"
```

Comment:
Using `fontSize="2xl"` will produce a fontSize of `24px` which is a size not defined in the DS for headings.

@konopkja @nloureiro since this is a new design, I'm wondering if we should go ahead and implement one of the predefined heading sizes from the DS. The closest sizes we have for this case is whether the `md` (20px) or `lg` (30px) size.

![image](https://github.com/ethereum/ethereum-org-website/assets/468158/d6a97a74-a2ea-41cf-9e2e-b4fdf5484693)

@corwintines if that is the case, then we could do something like this
```tsx
<Heading size="md"> // or `lg` whatever is decided
```

Reply:
hum.... I think I found out the problem. 

on the design system, we have 2 scales > 1 for heading and another for text

In the design proposal I used the node for text > 2xl and I probably should have used the headings scale > In this case you are right it's LG for the equivalent (Heading LG=30px on desktop and 24 on mobile)

I think we can change to heading LG